pkg/operation/botanist/component/nginxingressshoot/nginxingress.go

```diff
@@ -489,7 +505,7 @@ func (n *nginxIngress) computeResourcesData() (map[string][]byte, error) {
 				{
 					APIGroups:     []string{""},
 					Resources:     []string{"configmaps"},
-					ResourceNames: []string{"ingress-controller-leader-nginx"},
+					ResourceNames: []string{"ingress-controller-leader"},
```

Comment:
Changing the resource name will cause the existing running setup to break, and leader election will be required to be performed again.
```suggestion
					ResourceNames: []string{"ingress-controller-leader-nginx"},
```

Reply:
This is the name of the configmap created, we pass this here, https://github.com/gardener/gardener/blob/12518e2da78817ace8178c1afca928d15b9cc6e3/pkg/operation/botanist/component/nginxingressshoot/nginxingress.go#L719

Reply:
/approve